Tensile/Parallel.py

```diff
@@ -76,16 +75,18 @@ def ParallelMap(function: Callable, objects: Any, message: str="", enable: bool=
     Returns:
         A list containing the results of applying **function** to each item in **objects**.
     """
+
     from .Common import globalParameters
     from . import Utils
+
     threadCount = CPUThreadCount(enable)
+    message += f": {function.__name__}: {threadCount} thread(s)" + f", {len(objects)} tasks" if hasattr(objects, "__len__") else ""
```

Comment:
Add a verbosity flag to optionally print the function being called instead of printing every time.